cmd/web-handlers.go

```diff
@@ -972,6 +962,12 @@ func toWebAPIError(err error) APIError {
 			HTTPStatusCode: http.StatusForbidden,
 			Description:    err.Error(),
 		}
+	} else if err == errInvalidArgument {
```

Comment:
@harshavardhana could you add a comment on when this error will be seen? Could you provide a test to verify that we receive `errInvalidArgument` as expected?